common/datavalues2/src/columns/boolean/iterator.rs

```diff
@@ -12,6 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use std::sync::Arc;
+use std::sync::RwLock;
```

Comment:
`common_infallible::RwLock` or `parking_lot::RwLock`

Reply:
got it.